__tests__/app.test.js

```diff
@@ -191,8 +196,49 @@ describe('/api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('400 Bad Request: Invalid Id')
         })
     })
-})
+    test('POST 201: responds with the a named object containing the posted comment', () => {
+        const input = {
+            //Presumably the user would have to have an account before posting, so i used an already existing user
+            //Please let me know if this is correct in the feedback!
```

Comment:
Please remove inline comments. Use Github comments instead if necessary.

(Using the existing user is a good choice.)